### ISSUE TITLE
Controlling a 3D character

### ISSUE THREAD

**Issue by JeremyFerland** [Microsoft/RoomAliveToolkit#69]
hello,

is there a way to extract the skeleton data and apply them to a 3d character ?

thanks

**Comment by thundercarrot** [comment]
The Kinect server will stream skeleton data. But Kinect skeleton data is a collection of 3D points ("joints") not joint angles, which is what most character rigging schemes assume. If you look around I think you can find people dealing with how to go from Kinect's 3D points to joint angles.

**Comment by JeremyFerland** [comment]
do you plan to add this option to the toolkit ? in mean time i will try to merger the tool kit with Kinect v2 Examples with MS-SDK (https://www.assetstore.unity3d.com/en/#!/content/18708). Hop it will work

**Comment by thundercarrot** [comment]
Not really. I believe this is out of scope for the RoomAlive Toolkit. Sorry.
